html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <form action="http://www.example.com/login.php">
+    <label>Email:</label><br>
+      <input type="text" name="username" size="15" maxlength="30" />
+
+  </form>
+  <form action="http://www.example.com/profile.php">
+    <label> Why is coding cool?</label><br>
+    <select name="devices">
+      <option value="Select an option">Select an option</option>
+      <option value="Because it's interesting">Because it's interesting</option>
+      <option value="Because it's fun">Because it's fun</option>
+      <option value="Because it's powerful">Because it's powerful</option>
+    </select>
+  </form>
+  <form action="http://www.example.com/profile.php">
+    <label>
+      Favorite Language (choose one):
+    </label>
+      <br>
+      <input type="radio" name="genre" value="rock" checked="checked" /> HTML
+      <input type="radio" name="genre" value="pop" />
+      CSS
+      <input type="radio" name="genre" value="jazz" />
+      JavaScript
+
+  </form>
+  <form action="http://www.example.com/profile.php">
+    <label>
+      Would you like to sign up for more information?
+    </label>
+      <br>
+      <input type="checkbox" name="service" value="itunes" checked="checked" /> Sign me Up!
+
+  </form>
+  <form action="http://www.example.com/comments.php">
+    <label>
+      Comments
+    </label><br>
+    <textarea name="comments" cols="20" rows="4">Enter your comments</textarea>
+  </form>
```

Comment:
You don't need a form element for every form control element.  You want them to be in one form element.